static/js/compose_height.js

```diff
@@ -0,0 +1,52 @@
+import autosize from "autosize";
+import $ from "jquery";
+
+let is_full_size = false; // true or false
+
+function get_ele_height(selector) {
```

Comment:
We don't use abbreviations, so call this `get_element_height` if you need a function of this form.

Reply:
Sure, will take care next time. However, outdated as now not needed.